### ISSUE TITLE
checkbox field do not trigger change event. Custom onChange passed to props also doesn't work

### ISSUE THREAD

**Issue by pjarmalavicius** [erikras/redux-form#3369]
### Are you submitting a **bug report** or a **feature request**?
bug report

### What is the expected behavior?
Check / uncheck triggers "@@redux-form/CHANGE" action

### What's your environment?
7.0.3

**Comment by jokezer** [comment]
I have the same issue with radio input in 7.0.3.
When I click on radio input, I get 
```
@@redux-form/FOCUS
@@redux-form/BLUR
@@redux-form/FOCUS
```
And no change event.
Type="checkbox" works fine.

**Comment by jokezer** [comment]
I created a custom component for radio button, and it works fine without `onBlur` prop.

**Comment by svengau** [comment]
Same as @jokezer 

Here is the working code:
```
function MyCustomRadioInput(props) {
  const { input: { onBlur, ...rest } } = props;
  return <input type="radio" {...rest} />;
}
```

```
<Field
   name="..."
   component={MyCustomRadioInput}
   value="..."
/>

```

**Comment by likerRr** [comment]
I found extremely issue with checkboxe's `onChange` event. I bunlde my code via webpack for development and then serve it via `browser-sync`:

```
    browserSync.init({
        serveStatic: ['build'],
        server: {
          baseDir: `build/${projectType}`,
          middleware: [
            historyFallback(),
          ],
        },
      });
```

So when I serve it via browser-sync, event doesn't trigger, but if I serve it via simple node server - event starts work :) So I assume that browser sync adds some code which breaks my code. Has anyone faced such strange behavior?

**Comment by seberik** [comment]
@likerRr I observed the same for out code base. I could not find why onChange was not triggered for radio/checkbox inputs but the issue is not occurring in the compiled version.

**Comment by svengau** [comment]
Indeed, BrowserSync client seems to have a patch for IE to force change, which is not compatible with redux-form:

```
/**
 * Force Change event on radio & checkboxes (IE)
 */
utils.forceChange = function (elem) {
    elem.blur();
    elem.focus();
};
```

A quick and dirty fix: use browserSync.init({minify: false, ...), and patch the source with:

``` 
sed -i -- 's/bs.utils.forceChange(elem);/\/\/CODE REMOVED;/g' node_modules/browser-sync-client/dist/index.js
```

**Comment by seberik** [comment]
I disabled ghost-mode (https://browsersync.io/docs/options#option-ghostMode) and it seems to work.

**Comment by erikras** [comment]
There's a chance this might be fixed in `v7.1.0`, as it included a lot of fixes around checkboxes and `deepEqual()`. Can you confirm?

**Comment by seberik** [comment]
I could still reproduce the problem with redux-form v7.1.0 but I think that the problem, at leas for me is browser-sync.

**Comment by likerRr** [comment]
@seberik @erikras Can we somehow solve it with `browser-sync`'s team?

**Comment by Anandks1993** [comment]
@erikras I am also facing the same issue, for checkbox alone, validation is triggering only after onblur

**Comment by crystalena** [comment]
`ghostMode: false` did the trick, thanks @seberik

**Comment by dmitru** [comment]
For me the problem was caused by BrowserSync too.
So frustrating! But glad I found the solution - thanks. 👍

**Comment by wahlg** [comment]
I'm experiencing this issue with redux form 7.4.2, and it is not due to using browser-sync. The first time that a checkbox is unchecked, it does not dispatch the `@@redux-form/CHANGE` event, but on subsequent toggles it does.

**Comment by ArtemUstilov** [comment]
@wahlg did you fin solution?

**Comment by iamandrewluca** [comment]
Can someone provide a https://codesandbox.io with the issue? I tried but can't reproduce.
